stdlib/3/asyncio/exceptions.pyi

```diff
@@ -0,0 +1,17 @@
+import sys
+from typing import List, Optional
+
+if sys.version_info >= (3, 8):
+    class CancelledError(BaseException): ...
+    class TimeoutError(Exception): ...
+    class InvalidStateError(Exception): ...
+    class SendfileNotAvailableError(RuntimeError): ...
+    class IncompleteReadError(EOFError):
+        expected: Optional[int]
+        partial: bytes
+        def __init__(self, partial: bytes, expected: Optional[int]) -> None: ...
+    class LimitOverrunError(Exception):
+        consumed: int
+        def __init__(self, message: str, consumed: int) -> None: ...
+
+    __all__: List[str]
```

Comment:
`__all__` should should not be typed (event if existing stubs do this). If `__all__` exists, it has the same meaning as in implementation files. In this case it's not necessary.

Reply:
Thanks for correcting.